src/widget/wmainmenubar.h

```diff
@@ -54,6 +54,9 @@ class WMainMenuBar : public QMenuBar {
     void loadTrackToDeck(int deck);
     void reloadSkin();
     void rescanLibrary();
+#ifdef __DJINTEROP__
+    void exportLibrary();
+#endif
```

Comment:
Eventually we want to [remove the menu bar](https://bugs.launchpad.net/mixxx/+bug/1740365). Could you find somewhere in the library GUI to put this?

Reply:
The simplest idea that comes to mind is a Right Click -> Export on the "Tracks" and/or "Crates" items in the library sidebar.

Or maybe a button at the bottom of the sidebar.  Will experiment...

Reply:
I've now added some right-click functionality to the library sidebar.  The export dialog is now callable from three places:

* Right click on Tracks -> Export Library
* Right click on Crates -> Export All Crates
* Right click on a Crate -> Export to External Library

Hope this suffices.

Reply:
I think those are appropriate places.